feature-libs/cart/base/components/cart-shared/cart-item/cart-item.component.ts

```diff
@@ -22,7 +22,7 @@ export class CartItemComponent implements OnChanges {
   @Input() compact = false;
   @Input() item: OrderEntry;
   @Input() readonly = false;
-  @Input() quantityControl: FormControl;
+  @Input() quantityControl: UntypedFormControl;
```

Comment:
What is the plan to produce the migration material for all the breaking changes introduced in this PR? As per DoD, they should be included in this PR.

Reply:
We will ask customer to upgrade Angular first :)